### PR TITLE
Trying Bootstrap

### PR DESCRIPTION
**Pull request by bhskr44**
## Applying Bootstrap Framework in the Portfolio Project

Here is a list of components that we added using bootstrap:-
- Added navbar
- Added header section
- Implemented scrollspy component
- Added cards
- Added buttons
- Added modals
- Added forms
- Used d-flex for the layout

### CODE REVIEW

**Comment by AbhishekkM05** [comment]
dog-so-close
 You are so close !! You will get there!!
Good PR
Professional Commits

**Comment by ginabeki** [comment]
Looks good to me

**Comment by ginabeki** [comment]
download

Looks good to me

File: index.html [hunk 1]
```diff
@@ -0,0 +1,581 @@
...
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="css/main.css" />
+  </head>
+  <body>
+    <main data-bs-spy="scroll" data-bs-target="#navbar-example2">
+      <header>
+        <nav id="navbar-example2" class="navbar navbar-expand-lg px-5 fixed-top bg-white">
+          <div class="container-fluid">
+            <a class="navbar-brand" href="#">My Logo</a>
```
Comment (bhskr44) [inline index.html:24]: anchor tags with an invalid link will never solve the purpose of it. I would suggest using valid links inside \<a\> tags

**Review by bhskr44 — COMMENTED** [review]
Hi @bhskr44 🙋‍♂️,
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

unnamed


Highlights
Your project is professional ✅
Required Changes ♻️

**Pull request merged**